crypto/ed25519/ed25519.go

```diff
@@ -117,16 +151,19 @@ func GenPrivKeyEd25519FromSecret(secret []byte) PrivKeyEd25519 {
 
 var _ crypto.PubKey = PubKeyEd25519{}
 
+// PubKeyEd25519Size is the number of bytes in an Ed25519 signature
 const PubKeyEd25519Size = 32
 
-// Implements PubKeyInner
+// PubKeyEd25519 implements crypto.PubKey for the Ed25519 signature scheme
 type PubKeyEd25519 [PubKeyEd25519Size]byte
 
 // Address is the SHA256-20 of the raw pubkey bytes.
 func (pubKey PubKeyEd25519) Address() crypto.Address {
 	return crypto.Address(tmhash.Sum(pubKey[:]))
 }
 
+// Bytes marshals the PubKey using amino encoding.
+// Amino encoding was chosen due its portability
```

Comment:
not sure if it's the best place to talk about our reasoning

Reply:
I agree.